base/siteorigin-widget.class.php

```diff
@@ -568,7 +569,15 @@ function enqueue_scripts( $form_type = false ) {
 
 			wp_localize_script( 'siteorigin-widget-admin', 'soWidgets', array(
 				'ajaxurl' => wp_nonce_url( admin_url('admin-ajax.php'), 'widgets_action', '_widgets_nonce' ),
-				'sure' => __('Are you sure?', 'so-widgets-bundle')
+				'sure' => __('Are you sure?', 'so-widgets-bundle'),
+				'backup' => array(
+					'newerVersion' => __(
+						'There is a newer version of this widget\'s content available.',
```

Comment:
I suspect certain software may not be able to extract the string if it's broken up over multiple lines like that.